rest-server/templates/src/main/scala/com/actionml/templates/cb/CBDataset.scala

```diff
@@ -48,7 +52,8 @@ import com.mongodb.casbah.commons.conversions.scala._
   */
 class CBDataset(resourceId: String) extends Dataset[CBEvent](resourceId) {
 
-  lazy val store = new Mongo(m = "localhost", p = 27017, n = resourceId)
+  private lazy val config = ConfigFactory.load()
+  lazy val store = new Mongo(m = config.getString("mongo.host"), p = config.getInt("mongo.port"), n = resourceId)
   // Todo: need to get port from CLI or config
```

Comment:
Yes, MONGO_HOST & MONGO_PORT, these are the environment variables. By default localhost 27017